### PR TITLE
Extra Components(nav,toast,rating,grid)

### PR DESCRIPTION
**Pull request by varun-Tad**
I have  added four new components which include

1. Navigation
2. Toasts
3. Rating
4. Grid

Please review it

### CODE REVIEW

**Review by git-0r — CHANGES_REQUESTED** [review]
pr-review-20march

navigation overlapping content

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+i:hover {
+  color: var(--gold-color);
+}
+
+i {
```
Comment (git-0r) [inline Components/Rating/rating.css]: instead of selecting using 'i', you can use classes example-star, rating-star etc. to select the exact element.

**Comment by varun-Tad** [comment]
Thanks for the review @git-0r. Will look into it.

File: Components/Rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.fa-star:hover {
+  color: var(--gold-color);
+}
+
+.fa-star {
+  font-size: 30px;
+  cursor: pointer;
```
Comment (Ankur9669) [inline Components/Rating/rating.css:7]: Suggestion: Could give font-size in rems

File: Components/Rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.gold-color {
+  color: var(--gold-color);
+}
+
+.one-star-example {
+  font-size: 30px;
+}
```
Comment (Ankur9669) [inline Components/Rating/rating.css:16]: Same

File: Components/Rating/rating.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  fourStar.classList.add("gold-color");
+});
+
+fiveStar.addEventListener("click", () => {
+  oneStar.classList.add("gold-color");
+  twoStar.classList.add("gold-color");
+  threeStar.classList.add("gold-color");
+  fourStar.classList.add("gold-color");
+  fiveStar.classList.add("gold-color");
+});
```
Comment (Ankur9669) [inline Components/Rating/rating.js:35]: There is a lot of redundant code in this file, you can find other ways to acheive same target

File: Components/Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+.toast-resolve {
+  display: flex;
+  flex-direction: row;
+  margin-left: 40rem;
+}
+
+.toast-resolve p {
+  margin-right: 20px;
+  color: #bb86fc;
+}
```
Comment (Ankur9669) [inline Components/Toast/toast.css:49]: Suggestion: Use color variable here

**Review by Ankur9669 — APPROVED** [review]

File: Components/Toast/toast.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+.toast-resolve {
+  display: flex;
+  flex-direction: row;
+  margin-left: 40rem;
+}
+
+.toast-resolve p {
+  margin-right: 20px;
+  color: #bb86fc;
```
Comment (sri-rishi) [inline Components/Toast/toast.css]: Suggestion: use variable color naming for colors

File: Components/Toast/toast.css [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.toast-resolve {
+  display: flex;
+  flex-direction: row;
+  margin-left: 40rem;
+}
+
+.toast-resolve p {
+  margin-right: 20px;
```
Comment (sri-rishi) [inline Components/Toast/toast.css]: Suggestion: use `rem` for units for more than 1rem

File: Components/Toast/toast.css [hunk 8]
```diff
@@ -0,0 +1,49 @@
+.toast {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 30rem;
+  padding: 1.5rem;
+  border-radius: 0.5rem;
```
Comment (sri-rishi) [inline Components/Toast/toast.css:7]: Suggestion: can use `px` for units  for less than 1rem

**Review by sri-rishi — APPROVED** [review]
Code looks  good, feel free to merge them after changes

**Comment by varun-Tad** [comment]
Thanks for the review @Ankur9669  , @sri-rishi . Will make the changes

**Pull request merged**